openxla-nvgpu/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+cmake_minimum_required(VERSION 3.21...3.24)
+
+project(OPENXLA_NVGPU)
+
+# TODO: Fix this once the project is slotted into place.
```

Comment:
Nit: Id recommend making this required and failing if not set. But small one given this is common layout today, so feel free to ignore once we are further along but just add a comment in top level readme.

Reply:
I'll fix later. I'm not sure I really want this to be a cache option and need to fiddle with it.